code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -1255,6 +1255,8 @@
 		else
 			log_game("Cube ([monkey_type]) inflated, last touched by: NO_DATA")
 		var/mob/living/carbon/human/creature = new /mob/living/carbon/human(get_turf(src))
+		if(!fingerprintshidden)
+			fingerprintshidden = list()
 		if(fingerprintshidden.len)
```

Comment:
Easier way to fix this is just changing this line to `if(LAZYLEN(fingerprintshidden))`

Reply:
Changed